### DISCUSSION TITLE
Command for log out from the terminal

### DISCUSSION THREAD

**Discussion by gmr458** [hyprwm/Hyprland#1003] [Q&A] [answered]
In i3 I can use `i3-msg exit`, is there something similiar in Hyprland?, I created a simple menu with `tofi` to power off, restart and lock the screen, the only thing missing is the command to log out.
```sh
#!/bin/sh

if (ps -a | grep tofi >/dev/null); then
    killall -q tofi
else
	case $(printf "%s\n" "󰐥 Power Off" "󰑐 Restart" "󰖔 Suspend" "󰑻 Hibernate" "󰌾 Lock" "󰗽 Log Out" | tofi --config ~/.config/tofi/config_powermenu $@) in
		"󰐥 Power Off")
			systemctl poweroff
			;;
		"󰑐 Restart")
			systemctl reboot
			;;
		"󰖔 Suspend")
			systemctl suspend
			;;
		"󰑻 Hibernate")
			systemctl hibernate
			;;
		"󰌾 Lock")
			swaylock
			;;
		"󰗽 Log Out")
			# Command for logout
			;;
	esac
fi
```

**Comment by vaxerski** [comment, marked as answer]
`hyprctl dispatch exit 1`

**Reply by sai-kalai** [reply]
this exits the session, is there a way to "log out" like in the sense of KDE Plasma, conserving the open programs and being able to put the password back in and find everything? I'm not interested in functionality like e.g. sway's swaylock which shows an UI for inputting the password that is completely different from the display manager in use. Thanks for any help!

**Reply by sai-kalai** [reply]
My mistake, the behavior I describe is done by "lock", which shows a plasma-specific screen and not the dm; not "log out", which indeed goes back to the display manager, but closes all apps as well. The behavior I'm looking for is then done by hyprlock...
Is there a way to unify the GUI upon boot and locking?
